public/javascripts/notebook/job_progress.js

```diff
@@ -23,7 +23,7 @@ define([
       };
 
       // setup the progress chart
-      var svg = dimple.newSvg("#progress-bars", 100, 400);
+      var svg = dimple.newSvg("#progress-bars", 275, 175); // todo resize with panel → ref to panel needed and event from panel to be listened
```

Comment:
or maybe even better we'll just dumb the dimple-based progress bars as rubish... ;)